src/it/GetAndUnpackWithFileMapper/pom.xml

```diff
@@ -0,0 +1,40 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Build description -->
+    <groupId>com.googlecode.maven-download-plugin.it</groupId>
+    <artifactId>testBasic</artifactId>
+    <packaging>pom</packaging>
+    <version>${testing.versionUnderTest}</version>
+    <name>Test</name>
+
+    <!-- Build plugins and extensions -->
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>@project.groupId@</groupId>
+                <artifactId>@project.artifactId@</artifactId>
+                <version>@project.version@</version>
+                <executions>
+                    <execution>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>http://downloads.sourceforge.net/ant-contrib/ant-contrib-1.0b2-bin.zip</url>
+                            <unpack>true</unpack>
+                            <fileMappers>
+                                <org.codehaus.plexus.components.io.filemappers.RegExpFileMapper>
+                                    <pattern>README.txt</pattern>
```

Comment:
@mkarg technically, this pattern will match any of these names
- `README.txt`
- `README_txt`
- `READMEtxt`
..

I'd escape the dot to be sure it matches the dot only and not any char.
Mostly for readability ;)

Reply:
I can change that, but for the actualy outcome this makes no difference, as the `README.txt` file actually is the sole one matching that pattern.

Reply:
Sure, it's just for self documentation purpose )

Reply:
I can't see neiter benefit nor drawback (in fact it currently documents very well that wildcards can be used), so just did it in https://github.com/maven-download-plugin/maven-download-plugin/pull/206/commits/01c2c2c91a5e0548804e19b3ebc5c4b2283ede75 to fulfil your personal taste. ;-)

Reply:
@mkarg thank you, sir! :+1: